app/services/api/conversions/create_project_service.rb

```diff
@@ -33,4 +58,24 @@ def call
   rescue ActiveRecord::RecordInvalid
     raise ProjectCreationError.new("Failed to save user during API project creation, email: #{@project_params[:created_by_email]}")
```

Comment:
We should be careful here, the email is PII and this may leak that into the logs?